buildpack.md

```diff
@@ -4,7 +4,7 @@ This document specifies the interface between a single lifecycle and one or more
 
 A lifecycle is a program that uses buildpacks to transform application source code into an OCI image containing the compiled application.
 
-This is accomplished in four phases:
+This is accomplished in five phases:
```

Comment:
I am actually just going to back out all of the `buildpack.md` changes from the platform rewrite. If we fix this separately it will be easier from a branching/merging perspective.